_plugins/idp_translations.rb

```diff
@@ -0,0 +1,34 @@
+## This plugin copies the fonts and the images from the identity style guide
+## to the appropriate path on the built site
+##
+## The CSS and JS is already handled by Jekyll and the build:js npm script
+
+require 'fileutils'
+require 'uri'
+require 'net/http'
+require 'json'
+require 'open-uri'
```

Comment:
looks like we're using Net::HTTP instead of `URI.open` so we can prob remove this?
```suggestion
```